local-cli/server/findSymlinksPaths.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+const fs = require('fs');
+
+module.exports = function findSymlinksPaths(lookupFolder) {
+  const timeStart = Date.now();
+  const folders = fs.readdirSync(lookupFolder);
+  const timeEnd = Date.now();
+  console.log(`Scanning ${folders.length} folders in ${lookupFolder} (${timeEnd - timeStart}ms)`);
```

Comment:
Shiny! :D One final thought: you are only logging the directory read time; could you please include the symlink resolving in your timing as well? This is relevant too and may in fact be more costly than the simple read op.